extensions/datasketches/src/main/java/io/druid/query/aggregation/datasketches/theta/SketchMergeAggregatorFactory.java

```diff
@@ -66,7 +67,25 @@ public SketchMergeAggregatorFactory(
   @Override
   public AggregatorFactory getCombiningFactory()
   {
-    return new SketchMergeAggregatorFactory(name, name, size, shouldFinalize, isInputThetaSketch);
```

Comment:
yeah, don't remember why it got in this PR, but leaving it here anyway. this shouldn't cause any problems though.